content/influxdb3/cloud-dedicated/admin/account/_index.md

```diff
@@ -33,11 +33,28 @@ Your {{% product-name %}} account is a collection of {{% product-name omit="Clus
 
 ## View account information
 
-After you log in to the Admin UI, the Account Management portal displays the following information about your account:
+The Admin UI provides access to account information through the **All Accounts** page:
```

Comment:
this section is probably repetitive since we already have a dedicated section in admin-ui.md that explain this but I added it anyways to stay consistent with the old doc flow. Open to suggestions if this isn't preferred